src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs

```diff
@@ -44,7 +44,7 @@ public void BuildBasicApplicationReleaseProfiledAot ()
 				IsRelease = true,
 				AndroidEnableProfiledAot = true,
 			};
-			using (var b = CreateApkBuilder ("temp/BuildBasicApplicationRelease")) {
+			using (var b = CreateApkBuilder ("temp/BuildBasicApplicationReleaseProfiledAot")) {
```

Comment:
@radekdoulik There is no need to hard code these. You can use the `TestName` property. It will match the method name (and include/fix up arguments). That way if we add arguments or rename the test you only need to do it in one place. 

So use the following instead. 

```
using (var b = CreateApkBuilder (Path.Combine ("temp", TestName))) {
}
```

Reply:
Here is a sample https://github.com/xamarin/xamarin-android/blob/a360cb47023121dbb3ecbb9fab01cb345c5921fd/src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs#L228

Reply:
I already fixed that yesterday :-)